### ISSUE TITLE
--ignore-empty-strings=true flag fails for point datatypes during neo4j-admin database import

### ISSUE THREAD

**Issue by Praniyendeev** [neo4j/neo4j#13312]
I discovered that when I try to import csv with **:point{srid:9157}** column, and has an empty cell in that column. Then the terminal throws the following error.
```org.neo4j.internal.batchimport.input.InputException: ERROR in input
  data source: BufferedCharSeeker[source:/Users/Vengance/Library/Application Support/Neo4j Desktop/Application/relate-data/dbmss/dbms-4a6dd2eb-8ec0-4677-a9f7-1dddff5caf86/import/Neuprint_Neurons_315971.csv, position:365, line:1]
  in field: somaLocation:point{srid:9157}:15
  for header: [:ID(Body-ID),..... **very large header**] #somaLocation:point{srid:9157} is 15th  column
  raw field value: ??
  original error: Failed to parse map value: ''
```
**Neo4j Version:** 5.3.0 Desktop
**Operating System:** macOS
**API:** neo4j-admin import tool

### Steps to reproduce
1. Open terminal
2. Load a csv with column *something*:point{srid:9157} using neo4j-admin database import  with  the flag --ignore-empty-strings=true
3. make sure one of the rows in that column has an empty value

### Expected behavior
Ignore the empty value from the csv for any column

### Actual behavior
Ignores values for all other columns but **:point{srid:9157}**

**Comment by tinwelint** [comment]
Hi @Praniyendeev this is a known issue that was fixed in 5.4, so please try and upgrade.

**Comment by tinwelint** [comment]
Closing in advance. Feel free to open if issue still remains.
